docs/multicast-guide.md

```diff
@@ -0,0 +1,154 @@
+
+### Multicast NetworkPolicy statistics
+
+[The Antrea NetworkPolicyStats feature](feature-gates.md#networkpolicystats) also applies to multicast NetworkPolices.
+
+## Use case example
+
+[VLC](https://www.videolan.org/vlc/) is an open source multimedia player, it can be used for video streaming.
+
+In this section, we will show how multicast video steaming works in
+Antrea using VLC.
+
+To start a video streaming server, we start a VLC Pod to stream a sample video
+to the multicast IP address `239.255.12.42` with TTL 6.
+
+```bash
+kubectl run -i --tty --image=quay.io/galexrt/vlc:latest vlc-sender -- vlc --intf ncurses --vout dummy --aout dummy sample-video.mp4 --sout udp:239.255.12.42 --ttl 6 --repeat
+```
+
+You can verify multicast traffic is sent out from this Pod
+by running `antctl get podmulticaststats` in the `antrea-agent` Pod on the local Node,
+which indicates the VLC Pod is sending video streams out.
+
+You can also check the multicast routes on the Node by running command `ip mroute`,
+which should print the following route for forwarding the multicast traffic from
+the Antrea gateway interface to the transport interface.
+
+```bash
+ip mroute
+(pod-ip, 239.255.12.42)     Iif: antrea-gw0 Oifs: transport interfaces State: resolved
+```
+
+We also create a VLC Pod to be the receiver with the following command:
+
+```bash
+kubectl run -i --tty --image=quay.io/galexrt/vlc:latest vlc-receiver -- vlc --intf ncurses --vout dummy --aout dummy udp://@239.255.12.42 --repeat
+```
+
+It's expected to see inbound multicast traffic to this Pod by running
+`antctl get podmulticaststats` in the local `antrea-agent` Pod,
+which indicates the VLC Pod is receiving the video stream.
+
+Also, the `kubectl get multicastgroups` command will show that `vlc-receiver`
+has joined multicast group `239.255.12.42`.
+
+## Limitations
+
+This feature is currently only supported for IPv4 Linux clusters.
+Support for Windows and IPv6 will be added in the future.
+
+### Encap mode
+
+Multicast traffic from Pods to external hosts is not supported in encap mode.
+
+### Traffic in local network control block
+
+Multicast IPs in [Local Network Control Block](https://www.iana.org/assignments/multicast-addresses/multicast-addresses.xhtml#multicast-addresses-1) (224.0.0.0/24)
+should work only in Antrea encap mode. Multicast traffic destined for those addresses
+are not expected to be forwarded, therefore, no multicast route will be configured for them.
+External hosts are not supposed to send and receive traffic with those addresses either.
+
+### Linux kernel
+
+If the following situations apply to your Nodes, you may observe multicast traffic
+not routed correctly:
+
+1. Node kernel version under 5.4
+2. Node network doesn't support IGMP snooping
+3. Lots of inbound multicast traffic with groups that are not joined by the Pods on the local Node
```

Comment:
> busy networks with lots of unresolved
> multicast routing entries, the creation of new multicast group routes
> can be extremely slow and unreliable.

Here, routing entries are on the K8s Nodes, or some underlay switch/router? In any case, I feel no need to mention this one. It is a general issue with multicast. Listing it here without enough context can be misleading.

Reply:
Updated.I mean on the k8s Nodes. Remove this line.